src/style.css

```diff
@@ -1,3 +1,88 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --selected-link-color: #00acdc;
+
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
```

Comment:
These properties here might be organized a bit better (display followed by align and justify) but it's just a small suggestion